packages/core/package.json

```diff
@@ -58,5 +62,8 @@
     "build:watch": "nodemon --ignore dist --exec 'yarn build'",
     "prepublishOnly": "yarn clean && yarn build",
     "clean": "rm -rf dist"
+  },
+  "jest": {
+    "resolver": "jest-directory-named-resolver"
   }
```

Comment:
I don't think this will do anything, since jest won't be looking here for configuration.

I think we might have to add a jest config for each side because I think we'll have to set the [testing environment](https://jestjs.io/docs/en/configuration#testenvironment-string) to "jsdom" to be able to query the document modified by testing library on the web side.

Reply:
Good catch. Was an artifact that snuck in from first attempt. 

Removed.